src/read_text_file.cpp

```diff
@@ -0,0 +1,25 @@
+#include <fstream>
```

Comment:
I would prefer a plain C version if you're ok to port this this C.  Otherwise I can take care of this myself later on.  Using C strings has the potential to be ever so slightly faster too because we might avoid one allocation

Reply:
With this being the sole compiled file in storr, and given it contains the registration code, please rename to `storr.cpp` (or `storr.c` if you port it to C)

Reply:
Porting to C is fine with me. I actually did have a [pure C version in an earlier commit](https://github.com/wlandau/storr/blob/5607aa24f40f28b68229a73c4837bebb25db141c/src/read_text_file.cpp), which I can easily go back to. I became hesitant at the last minute because we would need to add extra guard rails against segfaults for key files that are empty, nonexistent, or too small.

By the way, do you have a preference between `fread()` or `fscanf()`? Maybe something else?

Reply:
See 4f06d03782e2d7bbe2ee7f35b97e0fcb893dac86. Uses `fgets()`.